src/components/Windows/Settings/setupSettings.ts

```diff
@@ -218,6 +218,24 @@ export async function setupSettings(settings: SettingsWindow) {
 		})
 	)
 
+	//Audio
+	settings.addControl(
+		new Toggle({
+			category: 'audio',
+			name: 'windows.settings.audio.volume.name',
+			description: 'windows.settings.audio.volume.description',
+			key: 'volume',
+			default: false,
+			onChange: async (val) => {
+				if (val) {
+					App.audioManager.masterVolume = 0
```

Comment:
The access of the audioManager now throws an error. I need to change this later anyways, you can uncomment the two lines for now.
![Bildschirmfoto 2021-05-21 um 18 48 10](https://user-images.githubusercontent.com/33347616/119171615-19aa8180-ba65-11eb-92d2-772096523988.png)